no-computer.md

```diff
@@ -11,41 +11,92 @@ _What need is met by your idea? Who are you helping? What is that person's
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+How else would you tell a story? All I want to do is change the granularity of the language used, from English words to mini stories. A language also makes sense here because it presents a great opportunity to design an extensible language: users can write and add their own mini stories to the set, and they can share their extensions with the community.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Funnily enough, the night before writing this response I was contemplating the
+idea of telling a story through a series of short, unrelated anecdotes. I find
+it fascinating that a deep character can be developed just by giving away brief
+flashes of their life, and I like the way the selection of which stories to
+tell has roughly as much impact on the big story as the content of the small
+stories does. But as interesting as this idea seems, I wondered when I would
+have time to try it.
+
+Today, as I was rewatching Victor's talk, desperately hoping that his ideas
+would stir something up in me, it just clicked. Victor perfectly describes the
+tragedy of lost ideas, and I realized that my idea could easily become one of
+them, just because of lack of time. And worse -- how many people with similar
+ideas can't get their ideas out? How many stories will be lost? Anything that
+might prevent that from happening excites me.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Big stories emerging from small.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The syntax of the language consists of a set of short, unrelated stories, somewhere between a paragraph and one to two pages. All the stories are told from the first person. The narrators are not necessarily the same, but not necessarily different (this means that consistency is important; details from the different stories cannot contradict each other). There should be stories told about various times in the narrator's life, from childhood to adulthood. The stories should cover a range of emotions and character traits, and they should range from hilarious to harrowing.
+
+These stories can be printed on cards, from which the creator can choose a subset to put in any order to create the life story of the narrator, told in chunks. Even though the stories are not related, I believe that the language will have fluency as an emergent property; the more one reads of the assembled story, the more one will feel as if they know the narrator. The character will be developed as much through the contents of the stories as through the creator's judgment of what the narrator should choose to reveal about him or herself.
```

Comment:
Just as you mentioned before, having a group of small stories about a character to define that character is interesting. That is often how we ourselves formulate our mental images of other people in the world (that is, through our small interactions with each other).